.github/workflows/mdformat.yml

```diff
@@ -12,8 +10,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
       - name: Mdformat
         uses: ydah/mdformat-action@main
+        if: contains(steps.changed-files.outputs.all_changed_files, '.md')
```

Comment:
memo) If we are going to go this far, perhaps we should simply remove filtering by `paths` or make it not required. WDYT?

Reply:
I suggest running this all the time with no path filtering, no checks if any files were changed etc, and keep it required.
Are there any downsides to this approach?

Reply:
That’s what I would suggest as well.

Reply:
It certainly sounds like it should always be done.
I will open the next PR and try to delete this one.
- https://github.com/rubocop/rubocop-capybara/pull/41